### DISCUSSION TITLE
in Prod i cannot open kimai anymore, no logs, in dev mode it works

### DISCUSSION THREAD

**Discussion by kruegge82** [kimai/kimai#4293] [Questions & Answers]
Hello,

today in one minute all is working, next minute:
"Service Unavailable
The server is temporarily unable to service your request due to maintenance downtime or capacity problems. Please try again later."

I am nothing chaned on the server.... after that im updating from 2.0.29 to 2.0.33 not helped
clear cache, not helped
error logs are writeable but no error
i changed prod to dev mode, site opens, but no errors in the error log, only "INFO" entries

In the Doktor all is green....

what can it be? can anyone help...
thanks Jahn

**Comment by kevinpapst** [comment]
I think it should fail in dev mode as well, but maybe:
https://github.com/kimai/kimai/issues/4246
https://github.com/kimai/kimai/blob/main/UPGRADING.md#2030

**Comment by kruegge82** [comment]
yes this helped, thank you very much...

**Comment by kruegge82** [comment]
Hello again,
the production is working for webbrowser, but the same error i get if i use the API,

DEV works
PROD not working

also no logs for errors....
maye also an idea?

for example i use this endpoint: /api/users?visible=1&term=39

database changed to "?charset=utf8mb4&serverVersion=mariadb-10.1.48"
im using mariadb-10.6.12 but if i change to this version nothing works anymore...
